services/subscriptions/handlers.go

```diff
@@ -9,11 +10,26 @@ import (
 // GetSubscriptionsHandler is the GET method handler for /subscriptions
 func (svc *SubscriptionService) GetSubscriptionsHandler() echo.HandlerFunc {
 	return func(c echo.Context) error {
-		response := SubscriptionResponse{
-			Value: svc.Subscriptions,
+		subsString, err := svc.Store.Get("subscriptions")
+		if err != nil {
+			panic(err)
```

Comment:
This is fine as long as we're vigilant about bubbling up human-readable errors that are obvious they came from Cerulean.